u6-spas/src/views/Profile.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="profile-edit">
+    <div>
+      <h1>{{ name }}</h1>
+      {{ tagline }} <br /><br />
+      <img :src="picUrl" width="30%">
+    </div>
+    <div>
+       <div>
+      <input type="text" name />
+      <input type="text" tagline />
+      <input type="text" picUrl />
```

Comment:
คุณลืม `v-model` ฉันค่อนข้างแน่ใจว่าวิธีนี้ใช้ไม่ได้ผล!
You forgot `v-model`. I'm pretty sure this doesn't work!